Vagrantfile

```diff
@@ -113,6 +115,5 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   # Set the name of the VM. See: http://stackoverflow.com/a/17864388/100134
-  config.vm.define vconfig['vagrant_machine_name'] do |d|
-  end
+  config.vm.define vconfig['vagrant_machine_name']
```

Comment:
I guess I'm asking the obvious, but will these two syntaxes result in the exact same behavior? (e.g. the machine name will still be set appropriately)